.gitpod.yml

```diff
@@ -0,0 +1 @@
+image: ghcr.io/notional-labs/cosmos
```

Comment:
what's wrong with the Tendermint-dev docker image?

Reply:
I should close this PR, because if we change to the notional one it should be done as a part of #278 

in 278 we would become incompatible with the tendermint-dev docker image.